package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hof-build",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
You don't need to do this, when you merge it will automatically patch update 😄

Reply:
Ah, excellent. Thanks for the info mate.